src/devices/awox.ts

```diff
@@ -97,6 +104,27 @@ const definitions: Definition[] = [
             'green', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up', 'brightness_move_down', 'brightness_stop',
             'recall_1', 'color_temperature_step_up', 'color_temperature_step_down'])],
     },
+    {
+        zigbeeModel: ['ERCU_3groups_Zm'],
+        fingerprint: [
```

Comment:
You can drop the fingerprint (identification goes via zigbeeModel)

Reply:
Did so. Hope I did it correctly :)

Reply:
But for some reason Github did not update the PR accordingly... I am a bit lost, using GitLab normally :D